river/tree/mondrian_tree.py

```diff
@@ -0,0 +1,53 @@
+from abc import ABC
```

Comment:
In terms of file structure, I would created a sub-folder called `mondrian` in the `tree` folder

Reply:
I added a "mondrian" tree folder and moved the scripts related to the Mondrian Trees into that one